services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -525,6 +525,51 @@ public long installOneSwitchFlow(DatapathId dpid, String flowId, Long cookie, in
         return pushFlow(sw, flowId, builder.build());
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<OFFlowMod> getDefaultFlows(DatapathId dpid) throws SwitchOperationException {
```

Comment:
I think you can split this method and make something like "installDefaultRUles()". 
That method just coll 7 methods (one method for each default rule). It's more readable